kubernetes/train.sh

```diff
@@ -12,6 +12,13 @@ while [ -n "${1-}" ]; do
     # curriculum script will handle copying the victim models for the predictor.
     --copy-initial-model) COPY_INITIAL_MODEL=1; ;;
     # Path to directory of TF weights for warmstarting.
+    # For an official KataGo model, download the TF weights from
```

Comment:
OK, but does this belong in this PR?

Reply:
this is porting documentation from `launch-job.sh`'s `--warmstart-ckpt` to here